scripts/run.sh

```diff
@@ -64,7 +64,9 @@ echo "Test pod exited with code:${podExitCode}"
 
 #Collect artifacts
 retrive-artifacts
+retrieveArtifactsExitCode=$?
 
 #Clean up
 delete_lease
-exit ${podExitCode}
+
+exit (( ${podExitCode}+${retrieveArtifactsExitCode} ))
```

Comment:
It was complaining about the spacing so I took it out. But yeah I think you
do.
On Jun 10, 2016 11:20 AM, "Seth Goings" <notifications@github.com> wrote:

> In scripts/run.sh
> <https://github.com/deis/e2e-runner/pull/10#discussion_r66649594>:
>
> >
> >  #Clean up
> >  delete_lease
> > -exit ${podExitCode}
> > +
> > +exit (( ${podExitCode}+${retrieveArtifactsExitCode} ))
>
> Don't you need a space around the + here?
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/deis/e2e-runner/pull/10/files/a71bfd84f3a3ec9dfe6b895252f502459bc06d1f#r66649594>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAaRGLweTzY2objBVdscQYn8uUKW96bGks5qKZzygaJpZM4IzG3r>
> .
>